File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    // 1. 정규표현식
    // let value = order.toString().match(/[369]/g) ?? []
    // return value.length;
    
    // 2. 
    // let answer = order.toString().split('').filter(v => v % 3 == 0 ? true : false)
    // return answer.length
    
    // 3
    const s = new Set('369')
    let answer = order.toString().split('').filter(v => s.has(v) ? true : false)
    return answer.length
}

File: 프로그래머스/0/120891. 369게임/369게임.py
def solution(order):
    # 1
    # answer = 0
    # order = str(order)
    # answer += order.count('3')
    # answer += order.count('6')
    # answer += order.count('9')
    
    # 2. map
    # order = str(order)
    # return sum(map(lambda x : order.count(x), '369'))
    
    # 3. filter
    # order = str(order)
    # return len(list(filter(lambda x: x in '369', order)))
    
    # 4. 정규표현식
    import re
    
    order = str(order)
    return len(re.findall('[369]', order))